src/components/locationsListing/query.ts

```diff
@@ -74,8 +74,8 @@ export const data: QueryData<
   const { data: allFacilities } =
     await fetchAndConcatAllResourceCollectionPages<NodeHealthCareLocalFacility>(
       RESOURCE_TYPES.VAMC_FACILITY,
-      queries
-        .getParams(RESOURCE_TYPES.VAMC_FACILITY)
```

Comment:
This previously used the `DrupalJsonApiParams` defined in the `params()` in the VAMC Facility query file, which is intended only for the full VAMC page, not the location listing blurb for the facility. As such, it added a _lot_ of unused includes.